packages/config/index.js

```diff
@@ -22,21 +22,24 @@ const getAppEntry = (rootFolder, isProd) => {
 };
 
 module.exports = (configurations) => {
+    const isProd = configurations.isProd || process.env.NODE_ENV === 'production';
     const { insights } = require(`${configurations.rootFolder}/package.json`);
     const gitBranch = process.env.TRAVIS_BRANCH || process.env.BRANCH || gitRevisionPlugin.branch();
-    const appDeployment = configurations.deployment || ((process.env.NODE_ENV === 'production' && betaBranhces.includes(gitBranch)) ?
+    const appDeployment = configurations.deployment || ((isProd && betaBranches.includes(gitBranch)) ?
         'beta/apps' :
         'apps');
 
     const publicPath = `/${appDeployment}/${insights.appname}/`;
-    const appEntry = getAppEntry(configurations.rootFolder, process.env.NODE_ENV === 'production');
+    const appEntry = configurations.appEntry || getAppEntry(configurations.rootFolder, isProd);
+    const chromePath = configurations.chromePath || require.resolve('@redhat-cloud-services/insights-standalone/package.json').replace('package.json', 'repos/insights-chrome-build');
```

Comment:
This is the one tricky thing that requires the frontend be run AFTER the backend. HtmlWebpackReplacePlugin needs to know where the local chrome dist is in order to grab the filename.